pkg/remote/resource_enumeration_error_handler.go

```diff
@@ -76,10 +122,18 @@ func handleAWSError(alerter alerter.AlerterInterface, listError *remoteerror.Res
 	return reqerr
 }
 
-func sendEnumerationAlert(provider string, alerter alerter.AlerterInterface, listError *remoteerror.ResourceEnumerationError) {
+func sendAlert(provider string, alerter alerter.AlerterInterface, listError *remoteerror.ResourceScanningError, p ScanningPhase) {
```

Comment:
I think it should be called `sendRemoteAccessDeniedAlert` since this method is scoped to `pkg/remote` it will prevent us eventual name collisions in the future.